### PR TITLE
add captureTime slider in logic analyser

### LINKED ISSUE #502: Add timebase slider in Logic Analyser
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
- The user does not have an option to choose between different time bases for logic analyser

* **What is the expected behaviour?**
- A timebase slider should be provided similar to an oscilloscope

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by priyanshunayan**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs hve been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [ ] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
Fixes #502 
I have added the timebase slider in logic analyzer. Earlier it was missing.

* **Does this PR introduce a breaking change?**
no

* **Preview / Steps to verify your work**:
slider

@AakashMallik

### CODE REVIEW

**Comment by AakashMallik** [comment]
@priyanshunayan Okay, there are a couple of changes that need to be done here. I'll list them here today.

**Comment by priyanshunayan** [comment]
@AakashMallik  I have done the following Changes.
1. Change timeBase to captureTime
2. Change unit from ms/div to ms
3. No X-axis change on dragging the slider.

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
         "babel-eslint": "^10.0.1",
         "concurrently": "^4.1.0",
         "devtron": "^1.4.0",
-        "electron": "^4.1.4",
+        "electron": "^4.2.10",
```
Comment (AakashMallik) [inline package.json]: Version updates should not be done like this. They have a proper discussion and a separate PR to go with. We don't know which feature might break with the update.

File: src/screen/LogicAnalyzer/LogicAnalyzer.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ const loadBalancer = window.require('electron-load-balancer');
 class LogicAnalyzer extends Component {
   constructor(props) {
     super(props);
+    this.captureTimeList = [0.1, 0.5, 1, 5, 10, 20, 25, 50, 100];
```
Comment (AakashMallik) [inline src/screen/LogicAnalyzer/LogicAnalyzer.js]: let's give the user an option of **[ 0.05,  0.1. 0.25, 0.5, 1, 2 ]**

File: package.json (hunk 1 above)
Comment (priyanshunayan) [reply package.json]: I am sorry. This was not intentional.

**Comment by priyanshunayan** [comment]
@AakashMallik Any updates?

**Comment by priyanshunayan** [comment]
@AakashMallik sir, is this PR going to work? Or please suggest some changes if there are any.

**Comment by NitinMadhukar** [comment]
@AakashMallik can i work on this?

**Comment by AakashMallik** [comment]
> @AakashMallik can i work on this?

Go ahead

**Comment by priyanshunayan** [comment]
@orangecms I have rebased onto the current development branch. Please let me know if there is any changes needed to done.

**Pull request merged**